fe/fe-core/src/main/java/com/starrocks/connector/delta/DeltaLakeStats.java

```diff
@@ -31,7 +31,11 @@ public class DeltaLakeStats {
     @SerializedName(value = "nullCount")
     public Map<String, Object> nullCount;
```

Comment:
Can you add some comments here to explain why this is a map and its key and value?
I guess it is column-name. If I was right, what will happen if column renaming is applied to a delta table?

Reply:
If you can add a comment for this class is even better, because I don't know how to use this class.
is this Stats table-level, or partition level or file-level? And please explain the difference from DelataLakeFileStats.

Reply:
This structure corresponds to the stats json of DeltaLake AddFile metadata, such as

```
"stats": "{\"numRecords\":1,\"minValues\":{\"c1\":1,\"c2\":11,\"c3\":\"111\"},\"maxValues\":{\"c1\":1,\"c2\":11,\"c3\":\"111\"},\"nullCount\":{\"c1\":0,\"c2\":0,\"c3\":0}}"
```

Reply:
DeltaLakeFileStats is the structure of the internal cache, and DeltaLakeStats is the structure corresponding to the original Json, I haven't thought of a good name to distinguish it at the moment.

Reply:
`DeltaLakeStats` is file level.

Reply:
This is just a structure obtained by deserializing metadata Json of add file, and there is no rename problem.

Reply:
Rename the class to `DeltaLakeAddFilesStatsProtocolStruct`